core/commands/dht.go

```diff
@@ -307,14 +307,14 @@ var provideRefDhtCmd = &cmds.Command{
 		ctx := notif.RegisterForQueryEvents(req.Context(), events)
 
 		go func() {
-			defer close(outChan)
 			for e := range events {
 				select {
 				case outChan <- e:
 				case <-req.Context().Done():
 					return
 				}
 			}
+			close(outChan)
```

Comment:
I'd be consistent and use this function here if we use it elsewhere.

Reply:
Done!